test/unit/messaging/messaging.spec.ts

```diff
@@ -2334,6 +2334,37 @@ describe('Messaging', () => {
       });
     });
 
+    const invalidImages = ['a', 'foo', 'image.jpg'];
+    invalidImages.forEach((image) => {
+      it(`should throw given an invalid image: ${ image }`, () => {
+        const message: Message = {
+          condition: 'topic-name',
+          android: {
+            notification: {
+              image,
+            },
+          },
+        };
+        expect(() => {
+          messaging.send(message);
+        }).to.throw('android.notification.image must be a valid URL string');
+      });
+    });
+
+    invalidImages.forEach((image) => {
```

Comment:
Lets move this somewhere else in this file. This section seems to be all about android config tests.